lib/commands/sdk.js

```diff
@@ -1368,15 +1379,52 @@ async function uninstall(logger, config, cli) {
  * @param {String} os - The name of the OS (osx, linux, win32)
  * @returns {Promise<Map<string, Release>>}
  */
-async function getReleases(config, os) {
-	const data = await fetch(urls.releases, config);
+exports.getReleases = async function getReleases(config, os) {
+	const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
```

Comment:
Do end users need to set this environment variable? Will gathering releases work without it?